classes/class-wc-connect-jetpack.php

```diff
@@ -110,30 +86,16 @@ public static function get_connected_user_data( $user_id ) {
 		 * @return WP_User | false
 		 */
 		public static function get_master_user() {
-			include_once ABSPATH . 'wp-admin/includes/plugin.php';
 			if ( self::is_active() && method_exists( 'Jetpack_Options', 'get_option' ) ) {
 				$master_user_id = Jetpack_Options::get_option( 'master_user' );
+
 				return get_userdata( $master_user_id );
 			}
```

Comment:
I'd recommend calling `Manager::get_connection_owner()` instead of pulling the data from options directly.